nurseconnect/models.py

```diff
@@ -0,0 +1,102 @@
+from django.contrib.auth.hashers import make_password, check_password
+from django.contrib.auth.models import User
+from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from nurseconnect.constants import GENDERS
+from molo.commenting.models import MoloComment
+from wagtail.contrib.settings.models import BaseSetting
+from wagtail.contrib.settings.registry import register_setting
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+
+
+class NurseConnectUserProfile(models.Model):
+    user = models.OneToOneField(
+        User,
+        related_name="nurse_connect_profile",
+        primary_key=True
+    )
+    gender = models.CharField(
+        max_length=1,
+        choices=GENDERS,
+        blank=True,
+        null=True
+    )
+    mobile_number = models.CharField(
+        max_length=15
+    )
+    staff_number = models.CharField(
+        max_length=15
+    )
+    facility_code = models.CharField(
+        max_length=15
+    )
+    security_question_1_answer = models.CharField(
+        max_length=128,
+        null=True
+    )
+    security_question_2_answer = models.CharField(
+        max_length=128,
+        null=True
+    )
+
+    # based on django.contrib.auth.models.AbstractBaseUser set_password &
+    # check_password functions (takes plain text, generates hash for database)
+    def set_security_question_1_answer(self, raw_answer):
```

Comment:
You take the security question answer, generate a hash for it, store it in the db, then when you have to answer the security question in the future, you'll then have to type what you think the answer is, it will then take that raw_answer, strip it of all trailing spaces, change it to lower case, and then check it against the hashed security answer that was stored. If it matches then it returns true. So its like a password checker.